tests/App/session_4.yml

```diff
@@ -0,0 +1,7 @@
+framework:
```

Comment:
use `include` to put the parameters compatible with all versions

Reply:
It's `imports`, not `include`: https://github.com/liip/LiipFunctionalTestBundle/commit/122d254c733b5cde2a29f566369cf23cc2283875